node/forest_libp2p/src/chain_exchange/message.rs

```diff
@@ -58,6 +58,22 @@ pub enum ChainExchangeResponseStatus {
     Other(i32),
 }
 
+#[cfg(test)]
+impl quickcheck::Arbitrary for ChainExchangeResponseStatus {
```

Comment:
`ChainExchangeResponseStatus` could use some refactoring but I think that is beyond the scope of this PR. Ideally it would have implementations for converting between `ChainExchangeResponseStatus` and `i32` in a type-safe manner. Right now it's possible to create `Other(0)` which is invalid. :-/